src/mixins/cartMixin.js

```diff
@@ -130,6 +74,30 @@ export default {
         },
       });
     },
+
+    createMyOrder() {
+      return this.$apollo.mutate({
+        mutation: gql`
+          mutation ($id: String!, $version: Long!, $locale: Locale!) {
+            createMyOrderFromCart(draft: { id: $id, version: $version }) {
+              ...OrderFields
+            }
+          }
+          ${ORDER_FRAGMENT}
+          ${MONEY_FRAGMENT}
+          ${ADDRESS_FRAGMENT}`,
+        variables: {
+          id: this.me.activeCart?.id,
+          version: this.me.activeCart?.version,
+          locale: this.$store.state.locale,
+        },
+        update: (store) => {
+          const data = store.readQuery({ query: BASIC_CART_QUERY });
+          data.me.activeCart = null;
+          store.writeQuery({ query: BASIC_CART_QUERY, data });
+        },
+      });
+    },
```

Comment:
Well and this feature sneaked in, from the checkout ;)
But if I remove it, I'll easily run into conflicts later, so I hope you forgive me 🙏

Reply:
haha I forgive you!